client/client_persistence_test.go

```diff
@@ -27,7 +27,7 @@ func TestPersistencePetraRobert(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), twoPartyTestTimeout)
 	defer cancel()
 
-	runTwoPartyTest(ctx, t, func(rng *rand.Rand) (setups []ctest.RoleSetup, roles [2]ctest.Executer) {
+	runAliceBobTest(ctx, t, func(rng *rand.Rand) (setups []ctest.RoleSetup, roles [2]ctest.Executer) {
```

Comment:
I think, the new name `runAliceBobTest` is misleading. Because, the tests is actually being run for `Petra` and `Robert` at one place and for `Alice` and `Bob` at another place.

`runTwoPartyTest` or `runTwoPartHappyTest` could be better names for this function.

Reply:
Reverting this change, because it apparently created other inconsistencies.